### PR TITLE
Add Binny's Beverage Depot

### LINKED ISSUE #4693: Add Binny's Beverage Depot
American liquor store chain. 

https://en.wikipedia.org/wiki/Binny's_Beverage_Depot
https://www.wikidata.org/wiki/Q2632011

### PR DESCRIPTION
**Pull request by mahihossain**

### CODE REVIEW

**Comment by mahihossain** [comment]
fixes #4693

**Comment by mahihossain** [comment]
close #4693

**Review by mahihossain — COMMENTED** [review]
close #4693

File: data/brands/shop/alcohol.json [hunk 1]
```diff
@@ -61,6 +61,17 @@
         "shop": "alcohol"
       }
     },
+    {
+      "displayName": "Binny's Beverage Depot",
+      "id": "binnysbeveragedepot-2740a5",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "brand": "Binny's Beverage Depot",
+        "brand:wikidata": "Q2632011",
```
Comment (bhousel) [inline data/brands/shop/alcohol.json:70]: Looks like [Q2632011](https://www.wikidata.org/wiki/Q2632011) just points to the generic wikidata for a "liquor store".  

What we really need is to lookup the Wikipedia/Wikidata for "Binny's Beverage Depot".  I did a quick Google search to find that it should be these:

```suggestion
        "brand:wikidata": "Q30687714",
        "brand:wikipedia": "en:Binny's Beverage Depot",
```

**Review by bhousel — CHANGES_REQUESTED** [review]

**Review by bhousel — APPROVED** [review]

**Comment by bhousel** [comment]
Thanks for the quick fix @mahihossain and welcome to the project 👋

**Comment by mahihossain** [comment]
> Thanks for the quick fix @mahihossain and welcome to the project 👋

@bhousel  Thanks a lot! You are very kind! And I'm really embarrassed about this silly mistake. I will do better next time.

**Pull request merged**